### ISSUE TITLE
Question about how to train this net

### ISSUE THREAD

**Issue by ran894694447** [andrearosasco/hyperpcr#1]
Hello!Can you tell me how to train this network? For example, which file to run or what environment to configure?

**Comment by andrearosasco** [comment]
Hi, sorry but I was busy with a demo and didn't really check the notifications here. Did you manage to run it?

**Comment by ran894694447** [comment]
> 

Thanks for your reply, I successfully trained the model via train.py.

**Comment by ran894694447** [comment]
Hi, I have one more question, when training the backbone of the network, did you use a pre-trained model? Because I noticed that in your paper the learning rate is 1e-4,but in your code the learning rate is 1e-5.I'm looking forward to your replay!

**Comment by andrearosasco** [comment]
Hi there, the learning rate is indeed 1e-4, here's the config: https://github.com/andrearosasco/confidence_shape_completion/blob/main/configs/main_config.py

**Comment by ran894694447** [comment]
Thanks for the reply, I will check it out！
